src/frontends/arcs/access_path.h

```diff
@@ -1,3 +1,18 @@
+//-----------------------------------------------------------------------------
+// Copyright 2022 Google LLC
```

Comment:
I'm going to bet that this file is from 2021, but maybe it was moved. Similar for everything under `src/frontend/arcs`

Reply:
Yep, they were moved. I can investigate each or just leave them with 2022?

Reply:
@markww Sorry, I'm still unsure if you'd like for me to investigate these files to work out which can be licensed to 2021 or if 2022 is acceptable.